### PR TITLE
[Fix]: Creating a navbar fixed at top 

### LINKED ISSUE #83: [Fix]: Creating a navbar fixed at top
## Description

Changing the ui of current Navabr.

## Screenshots (if applicable)

Actual:
old_navbar

Expected (highlight the part that requires changes):
(screenshot)
navbar
Keep the socials same as previous one and make it position: fixed

### PR DESCRIPTION
**Pull request by shreyvarshney1**
## Description
Changing the ui of current Navbar.

## Related Issues*
- Fixes #83 

## Checklist
- [x] I have forked the repo and pushed the changes according to contribution guidelines.
- [x] I have tested these changes locally.
- [x] I have tried to maintain the consistency of the code.
- [x] I have tested these changes locally.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: frontend/src/components/Navbar.jsx [hunk 1]
```diff
@@ -9,9 +9,9 @@ function Navbar() {
 
   return (
     <div
-      className={"w-full h-[8vh]  bg-main_theme "}
+      className={"w-full h-[12vh]  bg-main_theme fixed top-0 z-50 shadow-md"}
```
Comment (itsmohit097) [inline frontend/src/components/Navbar.jsx:12]: Keep h-[8vh]

File: frontend/src/components/Navbar.jsx [hunk 2]
```diff
@@ -9,9 +9,9 @@ function Navbar() {
...
     >
-      <div className="max-w-7xl mx-auto flex items-center justify-between lg:px-6  px-3 py-3">
+      <div className="max-w-7xl mx-auto flex items-center justify-between lg:px-6  px-3 py-5">
```
Comment (itsmohit097) [inline frontend/src/components/Navbar.jsx:14]: Keep py-5

**Review by itsmohit097 — COMMENTED** [review]
Make this changes
You jave to also change the menu list items: (All doctors, Specialities, Medicines, Appointment,Login in) 
Then on right side a cart icon have to be added and socials would be same.

**Comment by itsmohit097** [comment]
@shreyvarshney1 finish it by today positively.

File: frontend/src/components/Navbar.jsx (hunk 2 above)
Comment (shreyvarshney1) [reply frontend/src/components/Navbar.jsx:14]: If you are keeping h-[8vh] then py-1 should be there to make it look good @itsmohit097

**Comment by shreyvarshney1** [comment]
Check the Preview and let me know if any changes are required. I have tried best from my side! @itsmohit097

**Pull request merged**